components/lv-ts-tableprovider/LVSimple/src/test/resources/FileSystemRead-test.sh

```diff
@@ -0,0 +1,12 @@
+# Simple tests for FileSystemRead Table
+
```

Comment:
Shouldn't all script files in the resources dir have a copyright notice as in "ts-client"?

Reply:
I guess so -  I added a copyright to these test scripts.